added profille (#1)
amplication/.github: CONTRIBUTING.md

# Amplication Contribution Guideline

We're so glad that you're thinking about contributing to Amplication. We welcome your contributions!

Here are some guidelines and instructions for contributing to our projects. Please go through them before making a contribution.

## Code of Conduct

Amplication governs its participants according to the Contributor [Code of Conduct](https://github.com/amplication/amplication/blob/master/code_of_conduct.md). As a contributor, you agree to uphold this code. 

## FAQ and General Questions

> If your question concerns the code of a project, then you should create a [new issue] in the relevant project's repository.

Reviewer: It will be better if we can add a link to create a new issue here.

https://github.com/amplication/amplication/issues/new/choose

Author: Done

> You can also join amplication chat channel on discord and then ask your question(s) in the relevant channel of our [Discord](https://discord.com/invite/Z2CG3rUFnu.

Reviewer: Change the Discord link to https://amplication.com/discord

Author: Done

> You can also join amplication chat channel on discord and then ask your question(s) in the relevant channel of our [Discord](https://discord.com/invite/Z2CG3rUFnu.

Reviewer 2: ```suggestion
You can also join the Amplication server on [Discord](https://discord.com/invite/Z2CG3rUFnu and then ask your question(s) in the relevant channel.
```

Author: Done

> You can also checkout our [documentation](https://docs.amplication.com/docs/) for better understanding of amplication projects.

Reviewer: project*

Reviewer 2: ```suggestion
You can also check out our [documentation](https://docs.amplication.com/docs/) for a better understanding of Amplication.
```

Author: Done

## How to Contribute Code?

Bug fixes, performance improvements, code formatting, ...
There are a lot ways in which you can contribute code!

> The issues list of a project is a great place to find something that you can help us with.

Reviewer: it will be better if we can add issues link here: https://github.com/amplication/amplication/issues/

Author: Done
